### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by mschinis**
I did a fresh install today and ran across a couple of problems. This fixes all of the issues I had setting up the project.

1. **ember-cli**: Pin it down to minor version bumps. It was upgrading ember-cli to 2.13
2. **ember-load-initializers**: Added ember v2.12 support in v1.0.0
~~3. **ember-cli-fastboot**: Explained [here](https://github.com/ember-learn/ember-api-docs/issues/222)~~

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,9 @@ terraform.tfstate.backup
 .env
 .envrc
 
+# Editors
```
Comment (locks) [inline .gitignore]: 👀 could you put this in your global gitignore instead?

File: package.json [hunk 2]
```diff
@@ -50,7 +50,7 @@
     "ember-cli-deploy-gzip": "^0.2.4",
     "ember-cli-document-title": "^0.3.3",
     "ember-cli-eslint": "^3.0.2",
-    "ember-cli-fastboot": "^1.0.0-beta.15",
+    "ember-cli-fastboot": "1.0.0-beta.16",
```
Comment (locks) [inline package.json]: @kratiahuja suggested the latest beta. is it known not to work, or simply not tested?

Comment (kratiahuja) [inline package.json]: Please use `beta.19`. beta builds have no breaking change.

Comment (mschinis) [inline package.json]: 🤔 I remember there was a problem with beta 19, might be an ember-plugin or something. I'll have another look and update the PR.

File: .gitignore (hunk 1 above)
Comment (mschinis) [inline .gitignore]: yeap 👍

File: package.json (hunk 2 above)
Comment (kratiahuja) [inline package.json]: You might have upgraded to the rc builds which has the breaking change. Pin it to beta.19 and it should work.

**Comment by mschinis** [comment]
@kratiahuja updating to beta 19 worked, I think it may have been crashing before because of the loose definition of ember-cli which was grabbing a newer version.

@locks this is good to go

**Pull request merged**